src/main/asciidoc/release-notes.adoc

```diff
@@ -8,7 +8,7 @@
 
 == 1.9.0
 
-* The PlantUML diagram plugin now automatically selects GraphViz (prefered) or JDOT for rendering if not explicitly configured
+* The PlantUML diagram plugin now automatically selects GraphViz (prefered) or Smetana for rendering if not explicitly configured
```

Comment:
There should be a separate entry in the release notes for 1.12.0 that JDOT has been replaced by Smetana.

Reply:
True.